phpBB/config/default/routing/routing.yml

```diff
@@ -30,3 +30,6 @@ phpbb_report_routing:
 phpbb_ucp_routing:
     resource: ucp.yml
     prefix: /user
+
+phpbb_storage_routing:
+    resource: storage.yml
```

Comment:
Wouldn't it make sense to also add a prefix for this?

Reply:
yes, /download/

Reply:
Then I'd go for moving the prefix here.